Formula/jq.rb

```diff
@@ -21,8 +21,8 @@ class Jq < Formula
     depends_on "libtool" => :build
   end
 
-  depends_on "oniguruma"  # jq depends > 1.5
-  depends_on "bison" => :build # jq depends on bison > 2.5
+  depends_on "oniguruma" # jq depends > 1.5
+  depends_on "bison" => :build # jq depends on bison >= 3.0
```

Comment:
That would also make sense, as we are just building from a release tarball. I'm happy to change the PR to do that instead, would save a dependency.